process_pr.py

```diff
@@ -1109,20 +1149,27 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
     # to use information during the actual comment loop
     for comment in issue.get_comments():
         all_comments.append(comment)
-        if bot_cache or technical_comment:
-            continue
         if comment.user.login.encode("ascii", "ignore").decode() != cmsbuild_user:
             continue
         comment_msg = comment.body.encode("ascii", "ignore").decode() if comment.body else ""
         first_line = "".join([l.strip() for l in comment_msg.split("\n") if l.strip()][0:1])
         if re.match(ISSUE_SEEN_MSG, first_line):
```

Comment:
may be change this to 
```
if (not  already_seen) and re.match(ISSUE_SEEN_MSG, first_line):
```
to avoid running `re.match(ISSUE_SEEN_MSG, first_line)` once already seen message it found. There should not be more than one already seen messages and we are only interested in first one anyway